pkg/agent/openflow/pipeline_windows.go

```diff
@@ -281,5 +281,17 @@ func (c *client) hostBridgeUplinkFlows(localSubnet net.IPNet, category cookie.Ca
 			Cookie(c.cookieAllocator.Request(category).Raw()).
 			Done(),
 	}
+	// The packages received from uplink are forwarded to LOCAL interface by default.
```

Comment:
By LOCAL you mean the bridge interface? Probably say "bridge interface" to be clear.

Reply:
Maybe "OVS bridge local interface" since I see this is used in other antrea code.